src/basic/architecture.h

```diff
@@ -237,6 +238,9 @@ int uname_architecture(void);
 #  else
 #    error "Unrecognized riscv architecture variant"
 #  endif
+#elif defined(__arc__)
+#  define native_architecture() ARCHITECTURE_ARC
+#  define LIB_ARCH_TUPLE "arc-linux"
```

Comment:
Is this really right? for all other archs this is a triplet. 

We usually use these definitions:

https://wiki.debian.org/Multiarch/Tuples

ARC is not on there, but it appears to be this would have to be "arc-linux-gnu"? can you elaborate?